### PR TITLE
Update sandcastle tutorials to match changes on cesium.com

### PR DESCRIPTION
**Pull request by hpinkos**
- Use ion imagery in the imagery tutorial
- Replace the particle system example

This needs to be merged before the release so Sandcastle matches the changes we're deploying

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @hpinkos!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/update-tutorials/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Apps/Sandcastle/gallery/Imagery Layers.html [hunk 1]
```diff
@@ -28,22 +30,23 @@
...
-var layers = viewer.imageryLayers;
-var earthAtNight = layers.addImageryProvider(new Cesium.IonImageryProvider({ assetId: 3812 }));
-earthAtNight.alpha = 0.5;
-earthAtNight.brightness = 2.0;
+var layers = viewer.scene.imageryLayers;
+var blackMarble = layers.addImageryProvider(new Cesium.IonImageryProvider({ assetId: 3812 }));
+
+blackMarble.alpha = 0.5; // 0.0 is transparent.  1.0 is opaque.
+
+blackMarble.brightness = 2.0; // > 1.0 increases brightness.  < 1.0 decreases.
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/Imagery Layers.html]: Picky... but maybe a little too concise. 
`Values range from 0.0 to 1.0, where 0 is transparent and 1.0 is opaque.`?
`Values > 1.0 increase brightness and values < 1.0 decrease.`?

Comment (hpinkos) [reply Apps/Sandcastle/gallery/Imagery Layers.html]: Actually, I think I should remove these comments entirely.

**Comment by hpinkos** [comment]
@ggetz ready

**Comment by ggetz** [comment]
Thanks @hpinkos !

**Pull request merged**